gavel/controllers/admin.py

```diff
@@ -39,10 +39,14 @@ def admin():
         item_counts[l] = item_counts.get(l, 0) + 1
     viewed = {i.id: {a.id for a in i.viewed} for i in items}
     skipped = {}
+    absent = {}
     for a in annotators:
         for i in a.ignore:
             if a.id not in viewed[i.id]:
                 skipped[i.id] = skipped.get(i.id, 0) + 1
+    for i in items:
+        for a in i.absent:
+            absent[i.id] = absent.get(i.id, 0) + 1
```

Comment:
Notice that each time already has a list of judges in absent. To get the count we can simply do 
```
absent[i.id] = len(a)
``` 
and get rid of the inner loop entirely. The reason we needed an inner loop for `skipped` was because the ignore table was set up so that it has annotator (judge) as primary key and item (team) as secondary. It's the other way around for our `absent` table.